anitya/static/css/cnucnu.css

```diff
@@ -25,3 +25,7 @@
     padding-left: 2px;
     padding-right: 2px;
 }
+
+table input[type="text"] {
```

Comment:
As the commit says, it makes the input fields a little larger, allowing, for example, to see the placeholder entirely.